src/Benchmarks/Startup.cs

```diff
@@ -13,6 +14,7 @@ public class Startup
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc();
+            services.AddSingleton(typeof(IHttpContextAccessor), typeof(InertHttpContextAccessor));
```

Comment:
Sure, but there's no guarantee that we won't take more of a dependency on this as long as it exists.